### ISSUE TITLE
Computing VQ2 Score

### ISSUE THREAD

**Issue by avshalomman** [yonatanbitton/wysiwyr#2]
Hi and thanks for the great work :)
I'm trying to run the VQ2 pipeline, and can't find the code for calculating the "yes" score in BLIP-2 VQA, the code only gets the text output without getting the logits.

Are you planning on releasing this?

Thanks!

**Comment by yonatanbitton** [comment]
Hi, yes it's a limitation of the BLIP2 variant, we plan to work on data release & the end-to-end model, but it will take time. We will update. Thanks.
